packages/components/src/Shield/Shield.tsx

```diff
@@ -22,7 +21,12 @@ const Shield: React.FunctionComponent<ShieldProps> = ({ impact = 'N/A', hasLabel
     color: attributes.color,
   };
 
-  const badge = <Icon size={size}>{attributes.title === 'Unknown' ? <QuestionIcon {...badgeProps} /> : <SecurityIcon {...badgeProps} />}</Icon>;
```

Comment:
We might want to add a prop to show an icon and which icon, since there might be places where we actually do want to show an icon. 

The default behaviour should still remain the same.